alpaca/data/historical.py

```diff
@@ -18,18 +18,24 @@ def __init__(
         api_key: Optional[str] = None,
         secret_key: Optional[str] = None,
         raw_data: bool = False,
+        url_override: Union[BaseURL, str] = None,
     ) -> None:
-        """Instantiates a Historical Data Client.
+        """
+        Instantiates a Historical Data Client.
+
         Args:
-        api_key (Optional[str], optional): Alpaca API key. Defaults to None.
-        secret_key (Optional[str], optional): Alpaca API secret key. Defaults to None.
-        raw_data (bool, optional): If true, API responses will not be wrapped and raw responses will returned from methods. Defaults to False.
+            api_key (Optional[str], optional): Alpaca API key. Defaults to None.
+            secret_key (Optional[str], optional): Alpaca API secret key. Defaults to None.
+            raw_data (bool, optional): If true, API responses will not be wrapped and raw responses will be returned from
+              methods. Defaults to False.
+            url_override (Union[BaseURL, str], optional): If specified allows you to override the base url the client points
```

Comment:
Why also include BaseURL in the url_override?

Reply:
you know thats a good point. Doesn't really make any sense, I'll change to be just str :see_no_evil: